src/sdk/pynni/nni/hyperopt_tuner/hyperopt_tuner.py

```diff
@@ -93,6 +95,8 @@ def json2parameter(in_x, parameter, name=NodeType.ROOT):
             else:
                 if _type in ['quniform', 'qloguniform']:
                     out_y = np.clip(parameter[name], in_x[NodeType.VALUE][0], in_x[NodeType.VALUE][1])
+                elif _type == 'randint':
+                    out_y = int(parameter[name])
```

Comment:
not proper, waiting for @xuehui1991 's comments

Reply:
Now we use hp.randint (with upper only) as workaround.